components/LayerObject.vue

```diff
@@ -56,57 +61,110 @@
 </script>
 
 <template>
-    <ul class="layers">
+    <div class="empty-display" v-if="arrayZ.length === 0">
+        <img src="/public/Barista explaining6.png" alt="" />
+        <div class="empty-display_text">
+            <h1>No layers to display</h1>
+            <p>Start by adding an asset to the canvas.</p>
+            <button class="canvas-btn">Canvas</button>
+        </div>
+    </div>
+    <ul class="layers" :style="{ 'margin-top': navHeightPx }">
         <li
             v-for="(element, index) in arrayZSorted"
             :key="element.id"
             class="layer"
             :accessKey="element.id"
             @click="selectLayer(element.id, index)"
-            :style="{ border: index === selection ? '1px solid black' : 'none' }"
+            :style="{ border: index === selection ? `3px solid ${$primary}` : `1px solid ${$primary}` }"
         >
             <div class="asset-image">
                 <img class="img" :src="element.src" :alt="element.alt" />
             </div>
             <p class="layer-text">{{ element.alt }}</p>
             <div class="chevrons">
-                <div
-                    class="expand-less icon"
+                <button
+                    class="expand-less icon icon-btn"
                     :style="{ opacity: index > 0 ? 1 : 0.25, cursor: index > 0 ? 'pointer' : 'not-allowed' }"
                     @click="sendEmitFront(element.id, index)"
                 >
                     expand_less
-                </div>
-                <div
-                    class="expand-more icon"
+                </button>
+                <button
+                    class="expand-more icon icon-btn"
                     :style="{
                         opacity: index < arrayZSorted.length - 1 ? 1 : 0.25,
                         cursor: index < arrayZSorted.length - 1 ? 'pointer' : 'not-allowed',
                     }"
                     @click="sendEmitBack(element.id, index)"
                 >
                     expand_more
-                </div>
+                </button>
             </div>
         </li>
     </ul>
 </template>
 
 <style scoped lang="scss">
+    .empty-display {
+        width: fit-content;
+        height: fit-content;
+        display: flex;
+        flex-direction: column;
+        row-gap: $spacer-5;
+        margin-top: $spacer-9;
+        align-items: center;
+        position: absolute;
+        top: 50%;
+        transform: translateY(-50%);
+    }
+
+    .empty-display img {
+        z-index: 10;
+        max-width: 24vw !important;
+    }
+
+    .empty-display h1 {
+        color: $primary;
+    }
```

Comment:
There is a class h1 (up till h6). That properly styles h1 as defined also with different sizes for different breakpoints.